docs/manpage.rst

```diff
@@ -241,7 +241,7 @@ Options controlling ReFrame output
 .. option:: --report-file=FILE
 
    The file where ReFrame will store its report.
-   The ``FILE`` argument may contain the special placeholder ``{sessionid}``, in which case ReFrame will generate a new report each time it is run by appending a counter to the report file.
+   The ``FILE`` argument may contain the special placeholder ``{sessionid}``, in which case ReFrame will generate a new report each time it is run by appending a counter to the report file. By default, ReFrame will write to $HOME/.reframe/reports/run-report.json.
```

Comment:
You don't need the `By default, [...]` part. The defaults are never mentioned here. They are only mentioned in the configuration file reference, so as to avoid stale entries in the documentation.

Reply:
So you should change it in the configuration reference only.

Reply:
You still need to change the default in the documentation here: https://github.com/eth-cscs/reframe/blob/master/docs/config_reference.rst